subproject-lifecycle.md

```diff
@@ -0,0 +1,121 @@
+# Lifecycle of a subprojects
+
+## Prerequisites
+
+We recognize 2 options how a subproject can be created in OSG. Either this is a preexisting workstream that members want to transfer under SIG governance or it is a brand new workstream which a member thinks that it should be governed by a SIG in OSG.
+
+### Create a new subproject
+
+1. Establish a relation with a SIG that takes patronage over your subproject idea. This SIG will become subproject's owner.
+   - Either get a verbal agreement on a SIG meeting, documented in SIG meeting notes
+   - Or file an issue in open-services-group/community with subproject details and get a SIG lead approval on the issue
+2. Define and declare scope of the subproject and end goal
```

Comment:
Question: shouldn't we also expect the creation of GitHub repo(s) associated with the sub-project?

Otherwise, where would the respective OWNERS file(s) reside?